### ISSUE TITLE
NuGet packages not available for pre-release versions of 5.7

### ISSUE THREAD

**Issue by jdom** [microsoft/TypeScript#60489]
### Acknowledgement

- [x] I acknowledge that issues using this template may be closed without further explanation at the maintainer's discretion.

### Comment

I was looking to test the update Microsoft.TypeScript.MSBuild nuget package v5.7 (any prerelease) buy couldn't find them in nuget.org. Are pre-release versions no longer being published? Can we still expect the stable release to be published as nuget?

**Comment by joj** [comment]
Hi! We are still publishing. We had some issues with the release that are now resolved. We should have the nuget available very soon. Sorry about that! J

**Comment by HolgerJeromin** [comment]
There was [no nuget release](https://www.nuget.org/packages/Microsoft.TypeScript.MSBuild) for 5.7.3 and v5.8.0-beta :-(

**Comment by jdom** [comment]
Indeed, this is causing us issue, as we use both sources (npm and nuget) for tooling that helps our devloop, and not having versions align across both completely breaks incremental compilation

**Comment by jdom** [comment]
v5.7.3 is now published to nuget.org :) Thanks @joj !
